tasks.py

```diff
@@ -70,6 +71,8 @@ def lint(ctx: Context, check: bool = False, unsafe_fixes: bool = False) -> None:
     cmds = ["ruff", "check", "."]
     if not check:
         cmds.append("--fix")
+    elif os.getenv("GITHUB_ACTIONS"):
+        cmds.append("--output-format=github")
```

Comment:
If this env var is present we are running inside the Github Actions runner and should use the GithubAction output format.